internal/host/host.go

```diff
@@ -158,6 +159,16 @@ func (m *Manager) RegisterHost(ctx context.Context, h *models.Host) error {
 	})
 }
 
+func (m *Manager) RegisterInstalledOCPHost(h *models.Host, db *gorm.DB) error {
```

Comment:
Maybe it should be part of the state machine?

Reply:
i think that moving it to state machine is an overkill. Especially since moving host to installed is not done via statemachine , but by the UpdateInstallProgress in host.go

Reply:
but all the things related to states are inside state machine. if you start with it other people will do the same

Reply:
it's part of our standard.

Reply:
fixed